bedrock/firefox/templates/firefox/releases/beta-notes.html

```diff
@@ -5,3 +5,9 @@
 {% extends "firefox/releases/release-notes.html" %}
 
 {% set channel_name = 'Beta' %}
+
+{% block optimizely %}
+  {% if waffle.switch('firefox-releasenotes-optimizely') %}
+     {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
```

Comment:
Ah, you're right, of course. I was rushing and didn't think it through. Sorry bout that.